packages/playground/website/src/components/browser-chrome/index.tsx

```diff
@@ -20,6 +22,7 @@ export default function BrowserChrome({
 	showAddressBar = true,
 	toolbarButtons,
 	initialIsFullSize = false,
+	storage = 'browser',
```

Comment:
Why do we need this? The default option is none so it might be a better fit here.

Reply:
@bgrgicak Ah! I think I did that in some testing 😇 Please disregard that line change, glad you like the rest :)